README.md

```diff
@@ -1 +1,11 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Hello, my name is Sophia Egrari
+
+I just finished my second year at UChicago as a **sociology** major and am interested in possibly completing the data science minor, as well, and am hoping that CFSS will give me a solid introduction to basic programming skills and data visualization!
+
+#### A little bit about me: 
+* I am from Seattle, Washington 
```

Comment:
I recommend leaving a blank line between Markdown elements. Helps to avoid rendering issues in more complex documents